tests/integration/nmstatectl_test.py

```diff
@@ -16,10 +16,12 @@
 #
 
 import json
+import os.path
```

Comment:
`import os` should be enough.

Reply:
Actually `import os` is doing more, so `import os.path` is enough because the other os content is not needed. I do not see why it would be better to import everything, therefore I will keep it for now. I am fine with changing this if you convince me why `import os` is better.